server.js

```diff
@@ -2007,9 +2014,9 @@ cache(function(data, match, sendBadge, request) {
         }
         sendBadge(format, badgeData);
       } else if (info === 'wheel') {
```

Comment:
A local variable in an if–else is a great use case for `let`: it allows the programmer to be confident that the variable is not being used after the end of the block.